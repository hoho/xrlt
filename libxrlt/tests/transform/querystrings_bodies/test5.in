id:0, type:500, last:0, error: 0, data:%7B%22hello%22%3A%22world%22%7D
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
