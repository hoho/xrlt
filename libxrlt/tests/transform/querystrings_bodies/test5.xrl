<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">

    <xrl:querystring name="qs">
        <xrl:type>json</xrl:type>
        <xrl:success select="concat(/hello/text(), '-', /hello/text())" />
    </xrl:querystring>

    <xrl:response>
        <xrl:text>|</xrl:text>
        <xrl:value-of select="$qs" />
        <xrl:text>|</xrl:text>
    </xrl:response>

</xrl:requestsheet>
