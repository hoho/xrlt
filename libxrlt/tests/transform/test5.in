id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:1, type:2, last:0, error: 0, data:hello=wo%20r
id:1, type:2, last:0, error: 0, data:ld&this=is&this
id:1, type:2, last:1, error: 0, data:=IS;very=co%20ol&blah;=he;=ha
id:0, type:0, last:0, error: 0, data:
