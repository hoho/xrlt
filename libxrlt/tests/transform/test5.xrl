<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">
    <xrl:response>
        <xrl:include>
            <xrl:href>/test/href/1</xrl:href>
            <xrl:type>querystring</xrl:type>
       </xrl:include>
    </xrl:response>
</xrl:requestsheet>
