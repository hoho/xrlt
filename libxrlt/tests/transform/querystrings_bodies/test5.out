XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: |
XRLT_STATUS_CHUNK
chunk: world-world
chunk: |
XRLT_STATUS_DONE
